articles/introduction-to-scikit-learn-in-python/index.md

```diff
@@ -0,0 +1,297 @@
+### Introduction to Scikit-Learn in Python - Part 1
+
+The concept of Machine Learning has been blooming over the past few years, and more often than not, graduate students and industry professionals have made a career switch to Data Science or Machine Learning. An essential ingredient for establishing familiarity in this field is to know your libraries and dependencies. Your work's significant chunk goes towards having the right approach towards the problem and manipulating the dataset regarding your approach. This multi-part article introduces the reader to SciKit-Learn, a vital library used to build statistical models to make predictions.
+
+### Pre-Requisites
+
+The reader is expected to understand basic libraries like NumPy and Pandas, machine learning, and machine learning algorithms, including linear and logistic regression, support vector machines and decision trees, and boosting algorithms. For better understanding, the reader is advised to go through the following articles on [Python](https://www.w3schools.com/python/), [NumPy](https://www.section.io/engineering-education/introduction-to-numpy/), [Matplotlib](https://www.section.io/engineering-education/matplotlib-visualization-python/) and [SciPy](https://www.tutorialspoint.com/scipy/index.htm).
+
+### Table of Contents
+
+1. [Introduction](#introduction)
+2. [Installing the Scikit-Learn Library](#installing-the-scikit-learn-library)
+3. [Dataset Transformations using sklearn](#dataset-transformations-using-sklearn)
+4. [Scikit-Learn for Standardization](#scikit-learn-for-standardization)
+5. [Scikit-Learn for Normalization](#scikit-learn-for-normalization)
+6. [Scikit-Learn for Encoding Categorical Features](#scikit-learn-for-encoding-categorical-features)
+7. [Scikit-Learn for Filling Missing Values](#scikit-learn-for-filling-missing-values)
+8. [Conclusion](#conclusion)
+9. [Further Readings](#further-readings)
+
+
+### Introduction
+
+SciKit-Learn (often referred to as sklearn) provides a wide array of statistical models and machine learning. sklearn, unlike most modules, is written in Python and not in C. Although written in Python, sklearn's performance is attributed to its usage of NumPy for high-performance linear algebra and array operations. SciKit-Learn was written as a part of Google's Summer of Code project and has since made lives easier for thousands of Python centered data scientists worldwide. This part of the series focuses on introducing the library and focusing on one aspect - dataset transformations, an important and crucial step to go through before building a prediction model.
+
+### Installing the Scikit-Learn Library
+
+Scikit-Learn requires the following libraries to be pre-installed: NumPy, SciPy, Matplotlib, IPython, Sympy, and Pandas. Let's go ahead and install them from the terminal using pip (works only for Windows)
+
+\>> pip install numpy
```

Comment:
Format these commands using code formatting available in markdown.

Reply:
Fixed it in the recent commit.